packages/solana/lib/src/rpc/json_rpc_client.dart

```diff
@@ -1,25 +1,29 @@
 import 'dart:async';
 import 'dart:convert';
+import 'dart:io';
 
-import 'package:http/http.dart';
 import 'package:solana/src/exceptions/http_exception.dart';
 import 'package:solana/src/exceptions/json_rpc_exception.dart';
-import 'package:solana/src/exceptions/rpc_timeout_exception.dart';
 import 'package:solana/src/rpc/json_rpc_request.dart';
 
 class JsonRpcClient {
   JsonRpcClient(
     this._url, {
     required Duration timeout,
     required Map<String, String> customHeaders,
+    HttpClient? httpClient, // Optional client for proxy support.
   })  : _timeout = timeout,
-        _headers = {..._defaultHeaders, ...customHeaders};
+        _headers = {..._defaultHeaders, ...customHeaders},
+        _httpClient = httpClient ?? HttpClient();
 
   final String _url;
   final Duration _timeout;
   final Map<String, String> _headers;
+  final HttpClient _httpClient;
   int _lastId = 1;
 
+  static get http => null;
```

Comment:
Why do you need this?

Reply:
We don't and I will remove this.

Reply:
Resolved by 59a3069085a53e622a49231398187cb383b46c79